README.md

```diff
@@ -36,10 +46,21 @@ module ([`/example`](examples/)),
 ```
 
 ## Handling Blockstack sessions
-The current implementation of the Blockstack Android SDK uses the j2v8 engine and blockstack.js. 
+`BlockstackSession`s use a session store to persist their state. 
+
+The default implementation of `ISessionStore` uses the default shared preferences. If an apps needs 
+to use two `BlockstackSession`s then both should use the same session store instance. If two session stores 
```

Comment:
```
If two session stores 
are used then it could happen that for example the user is loged out in one session while still being logged in 
in the other session.
```

Seems to be something to watch out for and you have a misspelling, so a better way to say this:

```
Failure to use the same session store instance can cause situations where a user is logged out of one session while still being logged into the other.
```